test/svm/utils.ts

```diff
@@ -103,7 +179,7 @@ export const relayerRefundHashFn = (input: RelayerRefundLeaf | RelayerRefundLeaf
     const abiCoder = new ethers.utils.AbiCoder();
     const encodedData = abiCoder.encode(
       [
-        "tuple(uint256 leafId, uint256 chainId, uint256 amountToReturn, address l2TokenAddress, address[] refundAddresses, uint256[] refundAmounts)",
+        "tuple( uint256 amountToReturn, uint256 chainId, uint256[] refundAmounts, uint256 leafId, address l2TokenAddress, address[] refundAddresses)",
```

Comment:
The order wasn't matching the one in EVM tests.

Reply:
ye, this is the thing I was trying to say in the call on Tuesday: we should make them match within the rust/solidity code! there is no reason from them not to match, its just extra complexity. I would suggest we update the rust to match the solidity ordering, even with differently named fields

Reply:
Yes, makes sense @chrismaree. I will work in the svm changes in a subsequent PR 👍